src/geonovum/leafletfigures.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Module: geonovum/leafletfigures
+ * Makes figures scalable via zoom and pan function
+ */
+import L from "geonovum/deps/leaflet";
+import easyButton from "geonovum/deps/easy-button";
+
+export function run(conf, doc, cb) {
+  doc.respecIsReady.then(function() {
+    Array.from(doc.querySelectorAll("figure.scalable img")).forEach(function(
+      image
+    ) {
+      var container = image.parentElement;
+      var div = doc.createElement("div");
+      div.id = "leaflet-figure";
```

Comment:
Here you want: `div.classList.add("leaflet-figure")` ... I'm not sure what setting this ID or class does, however? Do you really need it?